cmd/agent/main.go

```diff
@@ -61,6 +61,9 @@ type GrpcProxyAgentOptions struct {
 	// Configuration for connecting to the proxy-server
 	proxyServerHost string
 	proxyServerPort int
+
+	// file contains service account authorization token for enabling proxy-server token based authorization
+	serviceAccountToken string
```

Comment:
Can you name it serviceAccountTokenPath, in consistent with kubeconfigPath?

Reply:
renamed